full/Dockerfile

```diff
@@ -208,11 +209,16 @@ RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree"
 LABEL dazzle/layer=tool-docker
 LABEL dazzle/test=tests/tool-docker.yaml
 USER root
+ENV TRIGGER_REBUILD=2
```

Comment:
I don't think you need to use the `TRIGGER_REBUILD` trick when you also modify any line between `LABEL dazzle/layer=tool-docker` and the next `LABEL dazzle/layer` (as you did below).